pyproject.toml

```diff
@@ -21,6 +21,7 @@ opentelemetry-propagator-jaeger = "^1.12.0"
 opentelemetry-exporter-otlp-proto-http = "^1.13.0"
 backoff = ">=1.0,<2.0"
 # Above: backoff is pinned to <2.0 because https://github.com/open-telemetry/opentelemetry-python/issues/2829
+types-requests = "^2.28.11.2"
```

Comment:
This should be a dev dependency, and it already is: https://github.com/kolonialno/troncos/blob/main/pyproject.toml#L37